src/backend/distributed/worker/worker_partition_protocol.c

```diff
@@ -17,6 +17,7 @@
 #include "postgres.h"
 #include "funcapi.h"
 
+#include <netinet/in.h>
```

Comment:
On 2016-03-30 03:09:03 -0700, Matthew Seaman wrote:
> > Hm, aren't these in arpa/inet.h per posix? We should probably include both.
> 
> Either netinet/in.h or arpa/inet.h would work fine according to the man
> page.  I chose netinet/in.h arbitrarily.

IIRC it's actually different on different systems :(